scripts/generate-dev-bundle.ps1

```diff
@@ -378,6 +379,10 @@ $npmToolArgs = @{
 }
 Add-NpmModulesFromJsBundleFile @npmToolArgs
 
+# Leaving these probably doesn't hurt, but are removed for consistency w/ Unix.
+Remove-Item $(Join-Path $dirLib 'package.json')
+Remove-Item $(Join-Path $dirLib 'package-lock.json')
```

Comment:
Why do we need to remove these files?

In fact, in a recent fix I discovered that these files don't need to be removed in par on what happens with Unix dev_bundle, for few issues with Cordova in Windows to be fixed. ([context](https://github.com/meteor/meteor/pull/13388))

Reply:
hmm... I'm going to bring these back